webapp/markdown.py

```diff
@@ -44,25 +45,59 @@ def __init__(self, *args, **kwargs):
 
 
 class DescriptionBlock(BlockLexer):
-    grammar_class = DescriptionGrammar
+    grammar_class = DescriptionBlockGrammar
 
     default_rules = [
-        "fences",
         "block_code",
         "list_block",
         "paragraph",
         "text",
         "newline",
     ]
 
-    list_rules = ("fences", "block_code", "list_block", "text", "newline")
+    list_rules = ("block_code", "list_block", "text", "newline")
+
+
+class DescriptionInlineGrammar(InlineGrammar):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Rewrite to respect this convention:
+        # https://github.com/CanonicalLtd/snap-squad/issues/936
+        self.code = re.compile(r"^(`)([\S ]+)\1")
 
 
 class DescriptionInline(InlineLexer):
-    def _process_link(self, m, link, title=None):
-        line = m.group(0)
-        if line[0] != "!":
-            return super()._process_link(m, link, title)
+    grammar_class = DescriptionInlineGrammar
+
+    # Removed rules: link, reflink
+    default_rules = [
+        "escape",
+        "inline_html",
```

Comment:
why have we added `inline_html`? Or is this not what I think it is?

Reply:
removed